core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -158,7 +158,11 @@ object TestUtils extends Logging {
   }
 
   def getBrokerListStrFromServers(servers: Seq[KafkaServer], protocol: SecurityProtocol = SecurityProtocol.PLAINTEXT): String = {
-    servers.map(s => formatAddress(s.config.hostName, s.boundPort(protocol))).mkString(",")
+    servers.map(svr => { 
```

Comment:
Style nitpick: instead of `x.map(y => { .... })`, you can do `x.map { y => }` (a little less noise)